src/main/webapp/js/common.js

```diff
@@ -245,8 +245,16 @@ function sortTable(oneOfTableCell, colIdx, comp, ascending, row) {
     var columnType = 0;
     var store = [];
     var $RowList = $('tr', $table);
+    var shouldConsiderToolTip = false;
```

Comment:
We can just collapse this whole part into

```
var shouldConsiderToolTipYear = comp && comp.toString.includes('instructorHomeDateComparator');
```

Also, adding the "year" in the naming helps to disambiguate exactly which part of the tooltip is being considered